pkg/fixtures/mongo.go

```diff
@@ -1,55 +0,0 @@
-package fixtures
-
-import (
-	"github.com/globalsign/mgo"
-	"github.com/stretchr/testify/suite"
-	"github.com/technosophos/moniker"
-)
-
-var names = moniker.NewAlliterator()
-
-// DefaultMongoURL is the default Mongo URL for testing
-const DefaultMongoURL = "127.0.0.1:27017"
-
-// Mongo is a text fixture for use with github.com/stretchr/testify/suite tests
-//
-// use it like this:
-//
-//	type MyTests struct {
-//		*fixtures.Mongo
-//	}
-//
-//	func RunMyTests(t *testing.T) {
-//		suite.Run(t, &MyTests{Mongo: New(DefaultURL)})
-//	}
-type Mongo struct {
-	suite.Suite
-	url    string
-	DBName string
-	DB     *mgo.Database
-}
-
-// SetupTest creates a new mongo connection and DB, and attaches it to a
-// session before each test executes
-//
-// This implements the SetupTestSuite interface
-func (m *Mongo) SetupTest() {
-	sess, err := mgo.Dial(m.url)
-	m.Require().NoError(err)
-	m.DB = sess.DB(m.DBName)
-}
-
-// TearDownTest drops the database that was created in SetupTest
-//
-// This implements the TearDownTestSuite interface
-func (m *Mongo) TearDownTest() {
-	m.Require().NoError(m.DB.DropDatabase())
-}
-
-// NewMongo creates a new Mongo test fixture
-func NewMongo(url string) *Mongo {
-	return &Mongo{
-		url:    url,
-		DBName: names.NameSep("-") + "-athens-testing",
-	}
-}
```

Comment:
would we want to leave this around for future integration-level tests? or are they replaced by the compliance tests?

Reply:
@arschles they are replaced by compliance tests. Basically, if the Mongo URL Env Var is set, then mongo db will get tested, otherwise mongo gets skipped. Same is the case for Minio (and soon all the other storages)

Reply:
Awesome, thanks for clarifying @marwan-at-work